src/VisualStudio/Core/Def/Implementation/VirtualMemoryNotificationListener.cs

```diff
@@ -21,6 +22,9 @@ namespace Microsoft.VisualStudio.LanguageServices
     [Export, Shared]
     internal sealed class VirtualMemoryNotificationListener : ForegroundThreadAffinitizedObject, IVsBroadcastMessageEvents
     {
+        // memory threshold to turn off full solution analysis - 200MB
+        private const long MemoryThreshold = 200 * 1024 * 1024;
```

Comment:
in 32bit process, VM max is 2GB, but MemLow message is fixed at 1.5GB which is 3/4. so I lowered it to 200MB.